app/models/order.rb

```diff
@@ -0,0 +1,18 @@
+class Order < ApplicationRecord
+  has_many :orders_line_items
+  has_many :line_items, through: :orders_line_items
```

Comment:
I had different ideas on how I wanted to approach tracking order data and settled on leveraging the data we'd already have in the database and referencing that instead of duplicating that data into new order columns or some other related construct.

As a bonus, we save a little bit on performance since we aren't writing all of that line item-related data to a new table. We're instead simply setting foreign keys on a join table.